public/app/features/dashboard-scene/scene/keyboardShortcuts.ts

```diff
@@ -146,9 +146,10 @@ export function setupKeyboardShortcuts(scene: DashboardScene) {
 export function withFocusedPanel(scene: DashboardScene, fn: (vizPanel: VizPanel) => void) {
   return () => {
     const elements = document.querySelectorAll(':hover');
-    const focusedGridElement = document.activeElement?.closest('[data-viz-panel-key]');
+    const focusedGridElement = document.querySelector(':focus-visible')?.closest('[data-viz-panel-key]');
```

Comment:
It is super weird that `activeElement` doesn't work here, has `document.hasFocus()` should return false regardless. Also, when testing around `:focus-visible` is supposed to not be found, but for some reason it is (When I try to get it in the console it doesn't find an element, which is expected).